odl/set/pspace.py

```diff
@@ -619,10 +619,34 @@ def __repr__(self):
 
         return '{!r}.element({})'.format(self.space, inner_str)
 
-    def show(self, *args, **kwargs):
-        """Display the parts of this vector."""
+    def show(self, indices=None, **kwargs):
+        """Display the parts of this vector graphically
+
+        Parameters
+        ----------
+        indices : index expression, optional
+            If a single index (``indices=0``) displays that part
+            If a slice, displays those parts (``indices=slice(None)``)
+            If `list` of indices, display those (``indices=[0, 1, 3]``)
+            If a tuple is given, interprets the first element as an index for
+            this product space vector and passes the rest on.
+            (``indices=np.s_[0, :, :]``).
```

Comment:
The specific handling tuples is standard in python, numpy considers these different:

```python
x = array...
x[[1, 2, 3]] # Get elements 1, 2 and 3
x[(1, 2, 3)] # Get the element at index (1, 2, 3)
```

We simply follow this convention.

With that said, I'll try to improve this.